src/layers/embedding.cpp

```diff
@@ -96,13 +128,13 @@ std::tuple<Expr /*embeddings*/, Expr /*mask*/> Embedding::apply(Ptr<data::SubBat
   //        more slowly
 
   auto batchEmbeddings = apply(subBatch->data(), {dimWidth, dimBatch, dimEmb});
-#if 1
+#if 0
+    // @TODO: this is dead code now, get rid of it
+    // experimental: hide inline-fix source tokens from cross attention
+    auto batchMask = graph->constant({dimWidth, dimBatch, 1},
+                                     inits::fromVector(subBatch->crossMaskWithInlineFixSourceSuppressed()));
+#else
```

Comment:
Can we just delete this?

Reply:
Yeah of course, this code is outdated as noticed. I've deleted. Do you want me to delete other places where there are unused code related to this? Namely the definition of function crossMaskWithInlineFixSourceSuppressed https://github.com/marian-nmt/marian-dev/blob/fe74576dc3e957f97b4842951fb6ee7ec52fa324/src/data/corpus_base.cpp#L564-L610
and
https://github.com/marian-nmt/marian-dev/blob/fe74576dc3e957f97b4842951fb6ee7ec52fa324/src/data/factored_vocab.h#L82-L89

Reply:
I would say yes, go ahead and remove. The cleaner this code with factors is, the better. Thanks

Reply:
done